docs/alexa-directives.rst

```diff
@@ -213,27 +213,124 @@ An APL Command is sent with the `alexaAPLCommand` key in your controller. Just l
 PlayAudio
 ---------
 
-`Alexa Documentation <https://developer.amazon.com/docs/custom-skills/audioplayer-interface-reference.html>`_
+`Alexa Documentation <https://developer.amazon.com/docs/custom-skills/audioplayer-interface-reference.html#play>`_
 
 
 .. code-block:: javascript
 
-    const voxa = require('voxa');
-    const { PlayAudio } = voxa.alexa;
+    function register(app) {
+      app.onState('someState', () => {
+        const url = 'http://example.com/example.mp3';
+        const token = '{}';
+        const offsetInMilliseconds = 0;
+        const behavior = 'REPLACE_ALL';
+        const playAudio = new PlayAudio(url, token, offsetInMilliseconds, behavior);
 
-    app.onState('someState', () => {
-      const playAudio = new PlayAudio(
-        'http://example.com/example.mp3',
-        '{}',
-        0,
-        'REPLACE_ALL'
-      );
+        return {
+          directives: [playAudio],
```

Comment:
I don't really like the fact that all other directives can be called directly using some key in the transition object, but the PlayAudio one actually requires one to import the directive class. Feels like that's something that should be looked into